FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -328,306 +327,333 @@ def _addCfgToCS( self, cfg ):
+    defaultGroupProperties = [NORMAL_USER]
+    defaultHostProperties = [TRUSTED_HOST, CS_ADMINISTRATOR,
+                             JOB_ADMINISTRATOR, FULL_DELEGATION,
+                             PROXY_MANAGEMENT, OPERATOR]
+
+    for section in (cfgPath('Registry'),
+                    cfgPath('Registry', 'Users'),
+                    cfgPath('Registry', 'Groups'),
+                    cfgPath('Registry', 'Hosts')):
+      if not centralCfg.isSection(section):
+        centralCfg.createNewSection(section)
 
     if adminUserName:
-      if not ( adminUserDN and adminUserEmail ):
-        gLogger.error( 'AdminUserName is given but DN or Mail is missing it will not be configured' )
+      if not (adminUserDN and adminUserEmail):
+        gLogger.error('AdminUserName is given but DN or Mail is missing it will not be configured')
       else:
-        for section in [ cfgPath( 'Registry', 'Users', adminUserName ),
-                         cfgPath( 'Registry', 'Groups', defaultGroupName ),
-                         cfgPath( 'Registry', 'Groups', adminGroupName ) ]:
-          if not centralCfg.isSection( section ):
-            centralCfg.createNewSection( section )
-
-        if centralCfg['Registry'].existsKey( 'DefaultGroup' ): #pylint: disable=unsubscriptable-object,no-member
-          centralCfg['Registry'].deleteKey( 'DefaultGroup' ) #pylint: disable=unsubscriptable-object,no-member
-        centralCfg['Registry'].addKey( 'DefaultGroup', defaultGroupName, '' ) #pylint: disable=unsubscriptable-object,no-member
-
-        if centralCfg['Registry']['Users'][adminUserName].existsKey( 'DN' ): #pylint: disable=unsubscriptable-object
-          centralCfg['Registry']['Users'][adminUserName].deleteKey( 'DN' ) #pylint: disable=unsubscriptable-object
-        centralCfg['Registry']['Users'][adminUserName].addKey( 'DN', adminUserDN, '' ) #pylint: disable=unsubscriptable-object
-
-        if centralCfg['Registry']['Users'][adminUserName].existsKey( 'Email' ): #pylint: disable=unsubscriptable-object
-          centralCfg['Registry']['Users'][adminUserName].deleteKey( 'Email' ) #pylint: disable=unsubscriptable-object
-        centralCfg['Registry']['Users'][adminUserName].addKey( 'Email' , adminUserEmail, '' ) #pylint: disable=unsubscriptable-object
+        for section in [cfgPath('Registry', 'Users', adminUserName),
+                        cfgPath('Registry', 'Groups', defaultGroupName),
+                        cfgPath('Registry', 'Groups', adminGroupName)]:
+          if not centralCfg.isSection(section):
+            centralCfg.createNewSection(section)
+
+        if centralCfg['Registry'].existsKey('DefaultGroup'):  # pylint: disable=unsubscriptable-object,no-member
+          centralCfg['Registry'].deleteKey('DefaultGroup')  # pylint: disable=unsubscriptable-object,no-member
+        centralCfg['Registry'].addKey(
+            'DefaultGroup',
+            defaultGroupName,
+            '')  # pylint: disable=unsubscriptable-object,no-member
```

Comment:
This pylint ignore goes by line, as it is no longer on the same line as addKey pylint reports it again.